data/json/items/armor/exosuit/exosuit_modules.json

```diff
@@ -0,0 +1,200 @@
+    "type": "TOOL_ARMOR",
+    "category": "exosuit",
+    "name": { "str": "exosuit gasoline ICE power supply", "str_pl": "exosuit gasoline ICE power supplies" },
+    "description": "Designed to attach to an exosuit's back, this is an internal combustion engine generator with an integrated UPS, fueled by gasoline.",
+    "weight": "20 kg",
+    "volume": "25 L",
+    "material": [ "qt_steel", "plastic" ],
+    "symbol": "#",
+    "looks_like": "1cyl_combustion",
+    "color": "dark_gray",
+    "ammo": "gasoline",
+    "flags": [ "FRAGILE", "CANT_WEAR", "WATERPROOF", "IS_UPS", "USE_UPS", "EXO_PSU", "ONLY_ONE" ],
+    "armor": [ { "encumbrance": 0, "coverage": 40, "covers": [ "torso" ] } ],
+    "pocket_data": [
+      {
+        "pocket_type": "MAGAZINE",
+        "description": "This is a tank for gasoline.  Holds up to 10 liters.",
+        "rigid": true,
+        "watertight": true,
+        "ammo_restriction": { "gasoline": 10000 }
+      }
+    ]
+  },
+  {
+    "id": "exo_back_fuelcell",
+    "type": "TOOL_ARMOR",
+    "category": "exosuit",
+    "name": { "str": "exosuit turbine power supply", "str_pl": "exosuit turbine power supplies" },
+    "description": "Designed to attach to an exosuit's back, this is an adaptive gas turbine generator with an integrated UPS, capable of using nearly any fuel type.",
+    "weight": "25 kg",
+    "volume": "25 L",
+    "material": [ "qt_steel", "plastic" ],
+    "symbol": "#",
+    "looks_like": "1cyl_combustion",
+    "color": "dark_gray",
+    "ammo": [ "gasoline", "diesel", "jp8", "avgas", "lamp_oil", "motor_oil", "propane", "conc_alcohol", "ammonia_liquid" ],
+    "flags": [ "FRAGILE", "CANT_WEAR", "WATERPROOF", "IS_UPS", "USE_UPS", "EXO_PSU", "ONLY_ONE" ],
+    "armor": [ { "encumbrance": 0, "coverage": 40, "covers": [ "torso" ] } ],
+    "pocket_data": [
+      {
+        "pocket_type": "MAGAZINE",
+        "description": "This is a tank for fuel.  Holds up to 5 liters.",
+        "rigid": true,
+        "watertight": true,
+        "airtight": true,
+        "ammo_restriction": {
+          "gasoline": 5000,
+          "diesel": 5000,
+          "jp8": 5000,
+          "avgas": 5000,
+          "lamp_oil": 5000,
+          "motor_oil": 5000,
+          "propane": 5000,
+          "conc_alcohol": 5000,
+          "ammonia_liquid": 5000
+        }
+      }
+    ]
+  },
+  {
+    "id": "exo_lense_module",
+    "type": "TOOL",
+    "name": { "str": "exosuit multi-lens zoom module" },
+    "description": "A series of strangely shaped hexagonal lenses that flutter about inside your helmet.  As the lenses shift things around you stay perfectly in focus no matter your eyesight or distance.  It can be mounted to an exosuit helmet.",
+    "weight": "500 g",
+    "volume": "250 ml",
+    "price": "10 kUSD",
+    "material": [ "lvl4ballisticglass", "qt_steel" ],
+    "symbol": "(",
+    "color": "yellow",
+    "flags": [ "CANT_WEAR", "FIX_NEARSIGHT", "FIX_FARSIGHT", "ZOOM", "EXO_HELMET_GADGET" ],
```

Comment:
should this module have sun glare protection or even Flash protection?

Reply:
The frame itself has both of those features while active, the module offers vision enhancement.